mgmt_cli_test.py

```diff
@@ -185,6 +189,37 @@ def test_backup_multiple_ks_tables(self):
         self.verify_backup_success(bucket=location_list[0].split(':')[1], cluster_id=backup_task.cluster_id)
         self.log.info('finishing test_backup_multiple_ks_tables')
 
+    def test_backup_location_with_path(self):
+        self.log.info('starting test_backup_location_with_path')
+        self.update_cred_file()
+        location_list = ['s3:manager-backup-tests-us/path_testing/']
+        manager_tool = mgmt.get_scylla_manager_tool(manager_node=self.monitors.nodes[0])
+        mgr_cluster = manager_tool.add_cluster(name=self.CLUSTER_NAME + '_bucket_with_path',
+                                               db_cluster=self.db_cluster,
+                                               auth_token=self.monitors.mgmt_auth_token)
+        self._generate_load()
+        try:
+            mgr_cluster.create_backup_task({'location': location_list})
+        except ScyllaManagerError as error:
+            self.log.info(f'Expected to fail - error: {error}')
+        self.log.info('finishing test_backup_location_with_path')
```

Comment:
What about restoring and verifying?

Reply:
What about verifying it actually used the location path provided?

Reply:
This test is supposed to fail, as the backup needs a dedicated bucket, and cannot use a relative path inside a bucket.
So it has no verification to do, only the error raised by Scylla Manager.

Reply:
ok.